.github/workflows/test-windows.yml

```
@@ -57,8 +57,8 @@ jobs:
     - name: Print build system information
       run: python3 .github/workflows/system-info.py
 
-    - name: python3 -m pip install setuptools wheel pytest pytest-cov pytest-timeout defusedxml
-      run: python3 -m pip install setuptools wheel pytest pytest-cov pytest-timeout defusedxml
+    - name: python3 -m pip install wheel pytest pytest-cov pytest-timeout defusedxml
+      run: python3 -m pip install wheel pytest pytest-cov pytest-timeout defusedxml
```

```suggestion
    - name: python3 -m pip install pytest pytest-cov pytest-timeout defusedxml
      run: python3 -m pip install pytest pytest-cov pytest-timeout defusedxml
```
If we're removing setuptools because it is in pyproject.toml, then we can remove wheel by the same logic.